crates/copilot/src/copilot_chat.rs

```diff
@@ -112,6 +118,7 @@ impl Model {
         match self {
             Self::Gpt4o => 64_000,
             Self::Gpt4 => 32_768,
+            Self::Gpt4_1 => 1_047_576,
```

Comment:
I am assuming the max token count here means the context window. So i have provided the one specified in the model card of 4.1